networkit/cpp/centrality/GedWalk.cpp

```diff
@@ -536,10 +536,10 @@ void GedWalk::run() {
     if (boundStrategy == BoundStrategy::spectral) {
         assert(alpha < 1 / static_cast<double>(sigmaMax));
     } else if (boundStrategy == BoundStrategy::geometric) {
-        assert(alpha < 1 / static_cast<double>(degInMax));
+        assert(alpha < 1 / degInMax);
     } else {
         assert(boundStrategy == BoundStrategy::adaptiveGeometric);
-        assert(alpha < 1 / static_cast<double>(degOutMax + degInMax));
+        assert(alpha < 1 / degOutMax + degInMax);
```

Comment:
Deleted parentheses change the formula and change `1` to `1.0`. Same for line 539.